test/web/helpers.js

```diff
@@ -0,0 +1,58 @@
+/*!
+ * Copyright (c) 2019-2020 Digital Bazaar, Inc. All rights reserved.
+ */
+
+import {TokenService} from 'bedrock-web-authn-token';
+import {AccountService} from 'bedrock-web-account';
+import {authenticator} from 'otplib';
+import {MemoryEngine} from 'bedrock-web-store';
+
+export const store = new MemoryEngine();
+const tokenService = new TokenService();
+const accountService = new AccountService();
+
+export async function login({email, password, totp}) {
+  const authResults = {};
+  const challenge = authenticator.generate(totp.secret);
+  authResults.totp = await tokenService.authenticate(
+    {type: 'totp', email, challenge});
+  authResults.password = await tokenService.authenticate(
+    {email, type: 'password', challenge: password});
+  authResults.login = await tokenService.login();
+  return authResults;
+}
+
+export async function createAccount({
+  email,
+  password,
+  short_name = 'session-test'
+}) {
+  // check to make sure the account does not already exist.
+  const exists = await accountService.exists({email});
+  if(exists) {
+    // if the account exists just return
+    return;
```

Comment:
I think the better pattern here is to throw a `DuplicateError` and write the caller to handle `DuplicateError`.

This does beg the question, what does `accountService.create` do already?  Maybe this existence check is not needed if `create` already throws a `DuplicateError`.

Reply:
this is for setting up test data, however it does mean the failure case is not throwing so I will add a `DuplicateError`.

Reply:
Addressed this here: https://github.com/digitalbazaar/bedrock-web-session/pull/9/commits/4e251232ae3d11ad00b1a4a4e74499e85c8084d0